exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitExtensionTest.java

```diff
@@ -37,6 +38,7 @@
   private static final TestKit.Builder defaultBuilder = TestKit.builder()
       .withService(TestServiceModule.class);
 
+  @Disabled
```

Comment:
If all tests are disabled, it makes sense to put at class level (revert changes for file -> reapply at class level).

Reply:
Corrected